DP03_04a_Introduction_to_Phase_Curves.ipynb

```diff
@@ -9,7 +9,7 @@
     "<br>\n",
```

Comment:
<p>Curious why the time estimate was removed here, did the query speed up since earlier? (But 30 seconds is relatively short so maybe this isn't necessary to note).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>It took longer yesterday, about 2 mins. So I took it out not to provide wrong time estimate. It's still unclear in general whether we want to provide execution time information. </p>